target/linux/ramips/image/mt76x8.mk

```diff
@@ -240,6 +240,15 @@ define Device/jotale_js76x8-32m
 endef
 TARGET_DEVICES += jotale_js76x8-32m
 
+define Device/kroks_kndrt31r19
+  IMAGE_SIZE := 14m
+  DEVICE_VENDOR := Kroks
+  DEVICE_MODEL := Rt-Pot mXw DS RSIM
```

Comment:
Again, Model does not match the device node name.

Reply:
This is what is exposed to Web UI I suppose, so I just kept what was in original commit here:
https://github.com/kroks-free/openwrt/commit/ba19321f71c680cbbec2da5d60ee5f9947a33eca#diff-bb176229946c2cb6b2c4899e91408e84b49fe2518003b294e577f8605145c220R187

Please reopen if we need the match anyway.

Reply:
Actually, I didn't find any presence of this in the UI but it listed in `menuconfig` at least.

Reply:
You can add something like this:
````
  DEVICE_ALT0_VENDOR := ASUS
  DEVICE_ALT0_MODEL := RT-N12+
  DEVICE_ALT0_VARIANT := B1
````
To give it two names.

Reply:
Done, thank you